radlab-ui/webapp/src/components/forms/DefaultCreateForm.tsx

```diff
@@ -75,14 +88,55 @@ const DefaultCreateForm: React.FC<IDefaultCreateFormProps> = ({
     return mergeAllSafe([initialFormData, defaultSettingVariables])
   }
 
+  const saveMailBoxCred = async (payload: ISecretManagerReq) => {
+    try {
+      await axios.post("/api/secret", payload)
+    } catch (error: any) {
```

Comment:
Can we put a proper type here? I believe it's some sort of AxiosErrorResponse

Reply:
`catch` clause accept only `any` or `unknown` type hence used `any` here